components/common/button/Button.tsx

```diff
@@ -0,0 +1,44 @@
+interface ButtonProps {
+  title: string;
+  items: keyof typeof itemVariants;
+  fullWidth: boolean;
+  height?: number;
+  font?: any;
+  onClick?: () => void;
+}
+
+const itemVariants = {
+  largeGreen:
+    'flex-center rounded-12 bg-brand-primary text-text-inverse hover:bg-it-hover active:bg-it-pressed',
+  largeWhite:
+    'flex-center rounded-12 bg-white text-brand-primary border-1 border-solid border-brand-primary hover:border-it-hover hover:text-it-hover active:border-it-pressed active:text-it-pressed',
+  largeWhiteSecond:
+    'flex-center rounded-12 bg-white text-text-default border-1 border-solid border-text-secondary',
+  largeWhiteSecondDanger:
+    'flex-center rounded-12 bg-status-danger text-text-inverse',
+  smallGreen:
+    'flex-center rounded-12 bg-brand-primary text-white hover:bg-it-hover active:bg-it-pressed',
+  smallNoneBg:
+    'flex-center rounded-12 text-brand-primary border-1 border-solid border-brand-primary hover:border-it-hover hover:text-it-hover active:border-it-pressed active:text-it-pressed',
+  floatingGreen:
+    'flex-center rounded-40 bg-brand-primary text-white shadow-floating hover:bg-it-hover active:bg-it-pressed',
+  floatingWhite:
+    'flex-center rounded-40 bg-bg-inverse text-brand-primary borer-1 border-solide border-brand-primary shadow-floating hover:border-it-hover hover:text-it-hover active:border-it-pressed active:text-it-pressed',
+};
+
+const heightClass = (height: number) => `h-${height}`;
+const fontClass = (font: string) => `font-${font}`;
+
+export default function Button(props: ButtonProps) {
+  const { title, items, fullWidth, height, font, onClick } = props;
+
+  return (
+    <button
+      className={`${itemVariants[items]} ${fullWidth ? 'w-full' : ''} ${height ? heightClass(height) : ''} ${font ? fontClass(font) : ''}`}
+      style={{ height }}
+      onClick={onClick}
+    >
```

Comment:
`clsx` 라이브러리를 사용해서 아래와 같이 축약할 수 있을 것 같아요~!
```suggestion
import clsx from 'clsx';

<button
  className={clsx(
    itemVariants[items],
    fullWidth && 'w-full',
    height && heightClass(height),
    font && fontClass(font)
  )}
  style={{ height }}
  onClick={onClick}
>
```
`clsx`나 `classnames` 같은 라이브러리를 사용하면, 조건부 클래스 적용을 간소화 시킬 수 있는 장점이 있어요!
팀 미팅 때 하나 정해서 사용하는 거로 하시죠!